### PR TITLE
AutoPort must be all lowercase for xml

### PR DESCRIPTION
**Pull request by shewensheng**
The AutoPort param on Graphics struct must be all lowercase letter for
libvirt's xml.

Signed-off-by: shewensheng <shewensheng@chinatelecom.cn>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/virt-launcher/virtwrap/api/schema.go [hunk 1]
```diff
@@ -963,7 +963,7 @@ type VideoModel struct {
 }
 
 type Graphics struct {
-	AutoPort      string          `xml:"autoPort,attr,omitempty"`
+	AutoPort      string          `xml:"autoport,attr,omitempty"`
```
Comment (dankenigsberg) [inline pkg/virt-launcher/virtwrap/api/schema.go:966]: In your commit message, would you explain why this is a must? What behavior is currently broken? And can it be automatically tested?

Comment (shewensheng) [reply pkg/virt-launcher/virtwrap/api/schema.go:966]: > In your commit message, would you explain why this is a must? What behavior is currently broken? And can it be automatically tested?

We can find graphics xml format at:
https://libvirt.org/formatdomain.html#graphical-framebuffers
the autoport param must be all lowercase.

Comment (dankenigsberg) [reply pkg/virt-launcher/virtwrap/api/schema.go:966]: Thanks. Please state these details in the commit message.

I am not a maintainer of this repository. However, if your company is willing to add tests for spice and own the code, you can propose a PR. By owning the code, I mean that if it breaks, you are responsible to fixing it. If you do not, the code would be dropped.

Also, you are welcome to add your company to https://github.com/kubevirt/kubevirt/blob/main/ADOPTERS.md

Comment (shewensheng) [reply pkg/virt-launcher/virtwrap/api/schema.go:966]: How to propose a PR?

**Pull request closed without merging**